lightning_examples/finetuning-scheduler/.meta.yml

```diff
@@ -14,6 +14,6 @@ description: |
   schedule. It uses Hugging Face's ``datasets`` and ``transformers`` libraries to retrieve the relevant benchmark data
   and foundational model weights. The required dependencies are installed via the finetuning-scheduler ``[examples]`` extra.
 requirements:
-  - finetuning-scheduler[examples]>=0.2.0
+  - finetuning-scheduler[examples]>=0.3.0
```

Comment:
```suggestion
  - finetuning-scheduler[examples]>=0.3.0
  - datasets<2.8.0  # todo: AttributeError: module 'datasets.arrow_dataset' has no attribute 'Batch'
```

Reply:
@speediedan maybe need an update also in `finetuning` :otter:

Reply:
good catch man! Delighted the CI issue was resolved. Looks like 1.9 is on the way so I'll ensure the tutorial is updated appropriately for that version as soon as I have a chance as well.